clang/test/SemaCXX/ms-constexpr-new.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_cc1 -fms-compatibility -fms-compatibility-version=19.33 -std=c++20 -verify=supported %s
```

Comment:
Judging by the failure https://lab.llvm.org/buildbot/#/builders/60/builds/15037, you might want to test for specific triples.

Reply:
We can just remove `__cdecl` altogether. I put it there to replicate Microsoft's STL. MSVC accepts that test case without `__cdecl` too.

Regarding other compiler error, it was fixed in 3ec6c72551846b8f4143c8c101a1a6203e85a2aa

Reply:
Opened #74932.